Library/Homebrew/cmd/update-report.rb

```diff
@@ -235,22 +236,25 @@ def output_update_report
       EOS
     end
 
-    return if new_repository_version.blank?
+    return if new_tag.blank? || args.quiet?
 
     puts
-    ohai "Homebrew was updated to version #{new_repository_version}"
-    Settings.write "latesttag", new_repository_version
-    if new_repository_version.split(".").last == "0"
+    ohai "Homebrew was updated to version #{new_tag}"
+    new_major_version, new_minor_version = (new_tag.split(".")[0, 2]).map(&:to_i)
+    old_major_version, old_minor_version = (old_tag.split(".")[0, 2]).map(&:to_i) if old_tag.present?
+    if old_tag.blank? || new_major_version > old_major_version || new_minor_version > old_minor_version
       puts <<~EOS
         More detailed release notes are available on the Homebrew Blog:
-          #{Formatter.url("https://brew.sh/blog/#{new_repository_version}")}
-      EOS
-    elsif !args.quiet?
-      puts <<~EOS
-        The changelog can be found at:
-          #{Formatter.url("https://github.com/Homebrew/brew/releases/tag/#{new_repository_version}")}
+          #{Formatter.url("https://brew.sh/blog/#{new_major_version}.#{new_minor_version}.0")}
```

Comment:
```suggestion
          #{Formatter.url("https://brew.sh/blog/#{new_tag}")}
```

is this equivalent?

Reply:
Not if `new_tag` = 3.2.3. If one upgrades from 3.1.4 to 3.2.3, the code will show both blog and changelog messages. The line highlighted allows for this.